.bazelversion

```diff
@@ -1 +1 @@
-3.5.0
```

Comment:
I am suprised this works, but if the tests are passing, that's awesome :D

Reply:
it breaks at `6.0.0` because docker rules can't support that, but seems fine locally!

Cloud build tests time out on the PR, but it looks like it's just because postgres flaked - will retry.